Apps/Sandcastle/CesiumSandcastle.js

```diff
@@ -94,12 +98,16 @@ require({
         when,
         LinkButton,
         defined,
-        getBaseUri,
         Resource,
-        Cesium
+        Cesium,
+        pako,
+        ClipboardJS
 ) {
     'use strict';
 
+    // attach clipboard handling to our Copy button
+    var clipboardjs = new ClipboardJS('.copyButton');
```

Comment:
:+1:

Reply:
The styling of the copy button needs better styling to actually be 👍.  As it is it's merely 😐.  

It should look like the clipboard.js example:
![image](https://user-images.githubusercontent.com/32828/37545042-984fabb2-293d-11e8-9a84-25acb0c43f25.png)

Reply:
The fact that Sandcastle uses an ancient version of Dojo probably doesn't help.  @emackey you are probably the only one that actually knows dojo well enough to improve this.  Feel free to open a PR into master if you want to take a stab at fixing it.

One day we'll get to rewrite Sandcastle.